keyboards/bop/rules.mk

```
@@ -0,0 +1,22 @@
+MCU = at90usb1286
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
```

And just to make sure, if this is a teensy++ 2.0, then this is the wrong bootloader.  In that case, you want: 
```suggestion
BOOTLOADER = halfkay
```

Otherwise, atmel-dfu will cover most of the avr dfu type bootloaders.

---

Ah got it, thanks